jaxb-api/src/main/java/jakarta/xml/bind/annotation/XmlSchemaType.java

```diff
@@ -64,8 +64,8 @@
  *     level </p>
  * <pre>
  *     package foo;
- *     &#64;javax.xml.bind.annotation.XmlSchemaType(
- *          name="date", type=javax.xml.datatype.XMLGregorianCalendar.class)
+ *     &#64;jakarta.xml.bind.annotation.XmlSchemaType(
+ *          name="date", type=jakarta.xml.datatype.XMLGregorianCalendar.class)
```

Comment:
```suggestion
 *          name="date", type=javax.xml.datatype.XMLGregorianCalendar.class)
```

I think the datatype sub-package is still in javax (part of JAXP, I think...).